pkg/resources/management.cattle.io/v3/globalrole/validator.go

```diff
@@ -121,51 +122,55 @@ func (a *admitter) Admit(request *admission.Request) (*admissionv1.AdmissionResp
 		return nil, err
 	}
 
-	// check for escalation separately between cluster permissions and global permissions to prevent crossover
+	var returnError error
+
+	// Validate the global and namespaced rules of the new GR
+	globalRules := a.grbResolver.GlobalRoleResolver.GlobalRulesFromRole(newGR)
+	returnError = errors.Join(returnError, validateRules(globalRules, false, fldPath))
```

Comment:
Nit: I don't think we need the "errors.Join" call here since returnErr will be nil, so we could just do:

```suggestion
	returnError = validateRules(globalRules, false, fldPath))
```

But I might be missing something on this library.